src/gdcdictionary/schemas/exposure.yaml

```diff
@@ -294,6 +294,12 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  exposure_duration_hrs_per_day:
+    description: >-
```

Comment:
_terms has exposure_duration_hrs_per_day -- should this be a ref?

Reply:
Yes, changed to ref.